include/nbl/video/IPhysicalDevice.h

```diff
@@ -126,17 +225,98 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
                 return static_cast<uint32_t>(core::min<uint64_t>(infinitelyWideDeviceWGCount,maxResidentWorkgroups));
             }
         };
-        const SLimits& getLimits() const { return m_limits; }
+        
+        struct SProperties
+        {
+            //--> VkPhysicalDeviceProperties:
+            APIVersion  apiVersion;
+            // uint32_t driverVersion;
+            // uint32_t vendorID;
+            // uint32_t deviceID;
+            E_TYPE      deviceType;
+            // char     deviceName[VK_MAX_PHYSICAL_DEVICE_NAME_SIZE];
+            // uint8_t  pipelineCacheUUID[VK_UUID_SIZE];
+            SLimits     limits;
+            // VkPhysicalDeviceSparseProperties    sparseProperties;
+
+            //--> VkPhysicalDeviceDriverProperties
+            E_DRIVER_ID driverID;
+            // char driverName[VK_MAX_DRIVER_NAME_SIZE];
+            // char driverInfo[VK_MAX_DRIVER_INFO_SIZE];
+            // VkConformanceVersion conformanceVersion;
+
+            //--> VkPhysicalDeviceIDProperties
+            uint8_t deviceUUID[VK_UUID_SIZE];
+            // uint8_t driverUUID[VK_UUID_SIZE];
+            // uint8_t deviceLUID[VK_LUID_SIZE];
+            // uint32_t deviceNodeMask;
+            // VkBool32 deviceLUIDValid;
```

Comment:
fill UUID with 0s on OpenGL, obviously LUID won't be valid

`deviceNodeMask`, make it look like its a single GPU